manifests/web_server.pp

```diff
@@ -5,15 +5,27 @@
 # @example
 #   include irida::web_server
 class irida::web_server (
-  String $irida_ip_addr = $ipaddress
+  Boolean $apache_use_ssl,
+  String  $cert_file_path,
+  String  $cert_key_file_path,
```

Comment:
Believe you need default values otherwise CI test will fail.  Probably default to false and empty strings for the rest.

Reply:
Added defaults, hope it works!